quickstarts/aws/aws-qldb/config.yml

```diff
@@ -3,22 +3,29 @@ name: aws-qldb
 description: |-
   ## What is AWS QLDB?
 
-  Create a cryptographically verifiable ledger database that's managed by one
-  group.
+  Amazon Quantum Ledger Database (AWS QLDB) is a fully managed ledger database that provides a transparent, immutable, and cryptographically verifiable transaction log. Proactively instrument AWS QLDB with the New Relic infrastructure monitoring agent to monitor the stability and reliability of AWS QLDB.
 
 
-  ### Get started!
+  ### AWS QLDB quickstart highlights
+  
+  The New Relic AWS QLDB quickstart includes the following features
+  Dashboards- Proactively monitor metrics like ledger samples, provider readIOs latency, ledgers, overview of ledger provider, etc.
```

Comment:
This seems like a typo, did you want a bullet here?

Reply:
So I'd like this to be a colon, but I'm not sure if that's possible with the code.

Reply:
@aliandersonsn oh yes, I think you can use `:` in the YAML as it's treated like markdown. We can test it out and see if it passes the checks.